src/Algebra/Graph.hs

```diff
@@ -232,9 +241,11 @@ instance Num a => Num (Graph a) where
     abs         = id
     negate      = id
 
+-- | `==` is a good consumer of both arguments.
```

Comment:
You use backticks in a few places and I think Haddock doesn't support them -- please double-check the generated docs. I think we need to switch to quotes `'`.

Reply:
Backquotes seem to be the syntax for infix operators. This is producing a good output for me.